src/Symfony/Bundle/FrameworkBundle/Resources/config/form_csrf.php

```diff
@@ -12,18 +12,28 @@
 namespace Symfony\Component\DependencyInjection\Loader\Configurator;
 
 use Symfony\Component\Form\Extension\Csrf\Type\FormTypeCsrfExtension;
+use Symfony\Component\Security\Csrf\DoubleSubmitCsrfTokenManager;
 
 return static function (ContainerConfigurator $container) {
     $container->services()
         ->set('form.type_extension.csrf', FormTypeCsrfExtension::class)
             ->args([
-                service('security.csrf.token_manager'),
+                service('form.type_extension.csrf.token_manager'),
                 param('form.type_extension.csrf.enabled'),
                 param('form.type_extension.csrf.field_name'),
                 service('translator')->nullOnInvalid(),
                 param('validator.translation_domain'),
                 service('form.server_params'),
             ])
             ->tag('form.type_extension')
+
+        ->set('form.type_extension.csrf.token_manager', DoubleSubmitCsrfTokenManager::class)
+            ->args([
+                service('request_stack'),
+                service('logger')->nullOnInvalid(),
+                param('form.type_extension.csrf.header_name'),
```

Comment:
I think this should be configurable on a per-form basis too.. as JS possibilties can vary inside a given app, depending on the front-end sections (or be non-existent)

Reply:
I'd prefer removing options than adding, unless there's a use case. Here, I don't see any. I'd wait for one to add any options.